DESCRIPTION

```diff
@@ -1,74 +1,77 @@
-                  https://github.com/ropensci/onboarding/issues/188"),
+           comment = "Laura reviewed the package for rOpenSci, see \n                  https://github.com/ropensci/onboarding/issues/188"),
     person("Jeffrey", "Hollister", role = "rev",
-       comment = "Jeffrey reviewed the package for rOpenSci, see 
-                  https://github.com/ropensci/onboarding/issues/188"))
+           comment = "Jeffrey reviewed the package for rOpenSci, see \n                  https://github.com/ropensci/onboarding/issues/188")
+  )
 Maintainer: Marco Sciaini <marco.sciaini@posteo.net>
 Description: Provides neutral landscape models (<doi:10.1007/BF02275262>,
-    <http://sci-hub.tw/10.1007/bf02275262>).  
-    Neutral landscape models range from "hard" 
-    neutral models (completely random distributed), to "soft" neutral models 
-    (definable spatial characteristics) and generate landscape patterns that are 
-    independent of ecological processes.
-    Thus, these patterns can be used as null models in landscape ecology. 'NLMR' 
-    combines a large number of algorithms from other published software for 
-    simulating neutral landscapes. The simulation results are obtained in a
-    spatial data format (raster* objects from the 'raster' package) and can,
-    therefore, be used in any sort of raster data operation that is performed 
-    with standard observation data.                                                                                                  
+    <http://sci-hub.tw/10.1007/bf02275262>).  Neutral landscape models
+    range from "hard" neutral models (completely random distributed), to
+    "soft" neutral models (definable spatial characteristics) and generate
+    landscape patterns that are independent of ecological processes.
+    Thus, these patterns can be used as null models in landscape ecology.
+    'NLMR' combines a large number of algorithms from other published
+    software for simulating neutral landscapes. The simulation results are
+    obtained in a spatial data format (raster* objects from the 'raster'
+    package) and can, therefore, be used in any sort of raster data
+    operation that is performed with standard observation data.
 License: GPL-3
-Encoding: UTF-8
-LazyData: true
-ByteCompile: true
+URL: https://ropensci.github.io/NLMR/
+BugReports: https://github.com/ropensci/NLMR/issues/
 Depends:
     R (>= 3.1.0)
-SystemRequirements: C++11
-Roxygen: list(roclets = c("rd", "namespace", "collate"))
-RoxygenNote: 7.1.1
 Imports: 
     checkmate,
     dplyr,
+    fasterize,
     RandomFields,
     raster,
+    Rcpp,
+    sf,
     spatstat.core,
     spatstat.geom,
-    igraph,
     stats,
-    tibble,
-    fasterize,
-    sf,
-    Rcpp
-URL: https://ropensci.github.io/NLMR/
-BugReports: https://github.com/ropensci/NLMR/issues/
+    tibble
 Suggests:
-    testthat,
-    covr,
+    ggplot2,
+    highcharter,
     knitr,
+    landscapemetrics,
```

Comment:
you might want to move all the deps to another DESCRIPTION field (Config/Needs/website) but at least {attachment} helps find all deps @marcosci